From 3f48296097a7abdf3cb3f5592f2290d2e640177b Mon Sep 17 00:00:00 2001
From: Eduardo Habkost <ehabkost@redhat.com>
Date: Tue, 6 Apr 2010 22:38:51 -0300
Subject: [PATCH 01/12] net: remove NICInfo.bootable field

RH-Author: Eduardo Habkost <ehabkost@redhat.com>
Message-id: <1270593533-23946-2-git-send-email-ehabkost@redhat.com>
Patchwork-id: 8379
O-Subject: [PATCH RHEL6 qemu-kvm 1/3] net: remove NICInfo.bootable field
Bugzilla: 561078
RH-Acked-by: Justin M. Forbes <jforbes@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=561078

It is just set by net_set_boot_mask() and never used. The logic for rom loading
changed a lot since this field was introduced. It is not needed anymore.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 net.c |    1 -
 net.h |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 net.c |    1 -
 net.h |    1 -
 2 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/net.c b/net.c
index 1e6abdd..7b48225 100644
--- a/net.c
+++ b/net.c
@@ -1206,7 +1206,6 @@ void net_set_boot_mask(int net_boot_mask)
 
     for (i = 0; i < nb_nics; i++) {
         if (net_boot_mask & (1 << i)) {
-            nd_table[i].bootable = 1;
             net_boot_mask &= ~(1 << i);
         }
     }
diff --git a/net.h b/net.h
index 16f19c5..991f0fa 100644
--- a/net.h
+++ b/net.h
@@ -132,7 +132,6 @@ struct NICInfo {
     VLANState *vlan;
     VLANClientState *netdev;
     int used;
-    int bootable;
     int nvectors;
 };
 
-- 
1.7.0.3

